hw/top_earlgrey/top_earlgrey_asic.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
```

Comment:
Added new fusesoc core file for the top_earlgrey_asic.sv. Previously, I was adding this to the chip_sim.core file for dv (which is in a separate hierarchy).